packages/js/e2e-environment/bin/docker-compose.js

```diff
@@ -90,6 +90,12 @@ if ( ! process.env.WORDPRESS_URL ) {
 	process.env.WORDPRESS_URL = testConfig.url;
 }
 
+// Allow overriding default services and settings configurations.
+if ( fs.existsSync( '../../../docker-compose.override.yml' ) ) {
+	console.log( 'Using `docker-compose.override.yml`.' );
+	dockerArgs.unshift( '-f', path.resolve('../../../docker-compose.override.yml' ) );
+}
+
```

Comment:
@BrianHenryIE Since we call docker-compose already I'm wondering if we could maintain compatibility with docker compose instead of adding this custom code.

Naively I haven't tested it but could you try instead setting `cwd` of spawnSync on this line https://github.com/woocommerce/woocommerce/pull/36478/files#diff-8aab0da24b5f7600c94b1f5cfd48dcd81b417567e00f1bd90dd118f1af6ea72aR102 to the root directory of the project and then docker should auto pick-up any override that you've set?

Otherwise isn't the solution to this, just to include a override file at the directory this script is run from instead of adding this conditional load of an override to this script?